sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,94 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+    defNum = 1
+    defLast = False
+    hasPrinted = False
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+    try:
+        defNum = int(args[-1])
+        defLast = True
+    except ValueError:
+        pass
+
+    if not defLast:
+        for i in args:
+            defURL += i
+            defURL += "%20"
+    else:
+        argsCopy = args.copy()
+        del(argsCopy[-1])
+        for i in argsCopy:
+            defURL += i
+            defURL += "%20"
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(defURL) as resp:
+            rawDef = await resp.text()
+    
+    try:
+        rawDef = rawDef[rawDef.index("<!--JC: actual definitions-->")-10:]
+    except:
+        embed = discord.Embed(title=":exclamation: The information you entered does not correlate to a Dictionary Entry.", color=0xFF0000)
+        await message.channel.send(None, embed=embed)
+        return
+
+    rawDefArr = rawDef.split("<!--JC: actual definitions-->")
+
+    soupObj = BeautifulSoup(rawDefArr[defNum], "html.parser")
+
+    try:
+        for tag in soupObj.find_all(class_="intro-colon"):
+            definition += tag.text
+
+            if not definition:
+                raise IndexError("The website has changed, please fix.")
```

Comment:
The for loop will run if there are any "intro-colon" tags. If not, the definition wouldn't exist. Since you put `if not definition` inside the for loop, then when it's there, there must be a "intro-colon" tag, and therefore the definition exists due to the statement previous.